src/resources/help/en_US/relnotes3.5.0.html

```diff
@@ -35,7 +35,7 @@
 <p><b><font size="4">General features added in version 3.5</font></b></p>
 <ul>
   <li>Added command recording for Annotation Objects.</li>
-  <li>General Feature 2</li>
+  <li>Added new surface rendering option using <a href="https://github.com/KhronosGroup/ANARI-SDK">ANARI</a>.</li>
```

Comment:
It seems very unsatisfying to do all this great work adding this way cool feature and having a mention here of it reduced to a single line in the release notes :cry:. @griffin28 I know you will be sharing about this in other venues as well, but we should at least also add a short blog article to `visit.llnl.gov`. I can help with that if need be.